tests/test_probability.py

```diff
@@ -54,7 +55,7 @@ def test_joint(backend):
 
 
 def test_independent(backend):
-    tb, _ = backend
+    tb, _ = pyhf.get_backend()
```

Comment:
this should not have been changed. We have to inject the backend this way for testing purposes (if the tests are in weird orders, the `pyhf.get_backend()` isn't properly reset for tests).